lld/ELF/SyntheticSections.cpp

```diff
@@ -1703,6 +1704,18 @@ void DynamicReloc::finalize(Ctx &ctx, SymbolTableBaseSection *symt) {
   isFinal = true; // Catch errors
 }
 
+size_t RelocationBaseSection::getSize() const {
+  size_t size = relocs.size() * entsize;
+  if (ctx.arg.emachine == EM_RISCV) {
+    for (const auto &reloc : relocs) {
+      if (reloc.type.v & INTERNAL_RISCV_VENDOR_MASK) {
+        size += entsize;
+      }
```

Comment:
Alright, I see why you split bit 8 and bit 9, so this will need a slight update with my scheme. I still think mine is slightly neater, even with this check.

Reply:
I ended up opting not to merge the internal bit with the others, as this is _not_ the only spot that would need updating. It could still be done if you feel strongly about it.